OpenSim/Common/Component.cpp

```diff
@@ -615,86 +615,27 @@ void Component::setParent(const Component& parent)
 
 std::string Component::getFullPathName() const
```

Comment:
There is a `getString()` method in place already. It was named this way to avoid the `toString()` since this originally derived from `Object`. I'm tempted to change it to `toString()` now that it does not derive from anything. @aseth1 @chrisdembia thoughts?